src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -319,17 +250,15 @@ $(XamlMetaDataProviderPch)
             <_MdMergeInputs Include="@(Midl)">
                 <WinMDPath>%(Midl.OutputDirectory)%(Midl.MetadataFileName)</WinMDPath>
             </_MdMergeInputs>
-            <_MdMergeInputs Include="@(CppWinRTStaticWinMDReferences)"/>
+            <_MdMergeInputs Include="@(CppWinRTStaticLibraryProjectWinMDReferences)"/>
             <_MdMergeReferences Remove="@(_MdMergeReferences)" />
             <_MdMergeReferences Include="@(CppWinRTDirectWinMDReferences)" />
-            <_MdMergeReferences Include="@(CppWinRTDynamicWinMDReferences)" />
+            <_MdMergeReferences Include="@(CppWinRTDynamicLibraryProjectWinMDReferences)" />
             <_MdMergeReferences Include="@(CppWinRTPlatformWinMDReferences)" />
-            <_MdMergeRefsDistinct Remove="@(_MdMergeRefsDistinct)" />
```

Comment:
mdmerge can't handle duplicate winmd files - why is this no longer needed?

Reply:
We copy the files to a separate metadata dir now.